src/Shout/SHRBTextStyler.class.st

```diff
@@ -1077,7 +1077,9 @@ SHRBTextStyler >> visitParseErrorNode: anErrorNode [
 { #category : #visiting }
 SHRBTextStyler >> visitPragmaNode: aPragmaNode [
 
-	aPragmaNode selectorParts with: aPragmaNode keywordsPositions do: [ :keyword :position | self addStyle: #pragma from: position to: position + keyword size ].
+	aPragmaNode selectorParts
+		with: aPragmaNode keywordsPositions
+		do: [ :keyword :position | self addStyle: #pragma from: position - 1 to: position + keyword size ].
```

Comment:
The -1 seems off. Is it because the given position is wrong (it should not be)? or is it because you want to style `<` (but this will fail to highlight `<    pragmaaaaaa     >`)? or something else and I need mode coffee?

The following methods on aPragmaNode might help:

* `left` `right` should correspond to the position oh opening and closing `<` `>`
* `keywordsIntervals` seems to exist to avoid doing math to compute the full span of each keyword

Reply:
Oh thanks! 

I'll add tests with those cases :)